src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { memo } from 'react';
+import cn from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = memo(({
+  people,
+}) => {
+  const { slug = '' } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
Create an array of headings and map through them

Reply:
I don't think it's a good point. Because it's worse for optimization(unnecessary loop for data, that doesn't change) and don't help for undersanding)